packages/cart/integration-tests/__tests__/services/cart-module/index.spec.ts

```diff
@@ -38,6 +38,62 @@ describe("Cart Module Service", () => {
       )
     })
 
+    it.only("should create a line item with prices in different shapes", async () => {
+      // TODO: Will be removed. This is here for demoing purposes
+      const [createdCart] = await service.create([
+        {
+          currency_code: "eur",
+          items: [
+            {
+              title: "one",
+              quantity: 1,
+              // proof of backward compatibility
+              unit_price: 100,
+            },
+            {
+              title: "two",
+              quantity: 1,
+              // raw price creation
+              unit_price: { value: "1234.1234" },
```

Comment:
q: why can't I just do `unit_price: 1234.1234`?

Reply:
You can, like the example belou that.
The BigNumber support all these formats:

```
unit_price = 1234.1234
unit_price = "1234.1234"
unit_price = {
  value: "1234.1234"
}
unit_price = {
  value: "1234.1234",
  precision: 18,
  currency: "DOGE
  exchange_rate: {
    "USD": "0.0753"
  }
}
```

then you always have the property `.numeric` and `.raw`
obviouly, when we have more properties than the value, we'll always set an object to it.

Reply:
This was just to demonstrate the different ways of setting the price. You can indeed do that